### ISSUE TITLE
Depreciation warning on import

### ISSUE THREAD

**Issue by nonhermitian** [Qiskit/qiskit-terra#2176]
### Information

- **Qiskit Terra version**: master
- **Python version**:
- **Operating system**:

### What is the current behavior?
```python
from qiskit import *
```
gives
```python
/Users/paul/Desktop/GitHub/qiskit-terra/qiskit/circuit/variabletable.py:10: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  from collections import MutableMapping
```

This should not happen.

**Comment by mtreinish** [comment]
I pushed the fix for this earlier today: https://github.com/Qiskit/qiskit-terra/pull/2174

**Comment by nonhermitian** [comment]
Solved by #2174
